PayloadPkg/OsLoader/LoadImage.c

```diff
@@ -90,10 +91,15 @@ GetBootImageFromRawPartition (
     }
 
     DEBUG ((DEBUG_INFO, "Load image from SwPart (0x%x), LbaAddr(0x%llx)\n", SwPart, LbaAddr));
-    Status = GetLogicalPartitionInfo (SwPart, LoadedImage->HwPartHandle, &LogicBlkDev);
-    if (EFI_ERROR (Status)) {
-      DEBUG ((DEBUG_INFO, "Get logical partition error - %r\n", Status));
-      return Status;
+    if (SwPart != 0xFF) {

```

Comment:
In InitBootFileSystem(), do you think we should skip InitFileSystem() if SwPart is 0xFF ?   
In this way, we can save boot time.

Reply:
Looks like we will already skip if FsType is set to RAW (FsType < EnumFileSystemMax). Do you think we should still add SwPart check too?

Reply:
ahh, you are right.  :)
Please ignore this comment then.